docs/repo/loading-components-with-projects.md

```diff
@@ -0,0 +1,25 @@
+## Loading components
+
+Components that are exported through MEF will be automatically discovered as necessary by any imports that exist, but loading and initializing is still a manual process. Additionally, components are typically applied to specific capabilities and due to the way SDK projects work these can be dynamic, which come and go throughout the lifetime of a project, when:
```

Comment:
What do you mean by "due to the way SDK projects work"?

Reply:
Will revisit